content/actions/guides/building-and-testing-python.md

```diff
@@ -399,15 +399,21 @@ jobs:
 
 You can configure your workflow to publish your Python package to any package registry you'd like when your CI tests pass.
 
-You can store any access tokens or credentials needed to publish your package using secrets. The following example creates and publishes a package to PyPI using `twine` and `dist`. For more information, see "[Creating and using encrypted secrets](/github/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)."
+In this example, your package is uploaded to PyPI each time you [publish a release](https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository).
+Publishing a pre-release will upload a package to the development version of PyPI.
+
+You can store any access tokens or credentials needed to publish your package using secrets.
+In this example, you will need to create two [PyPI API tokens](https://pypi.org/help/#apitoken).
+For more information, see "[Creating and using encrypted secrets](/github/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)."
+The Python Packaging Authority (PyPA) also has a [guide to Github Actions](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows).
 
 {% raw %}
 ```yaml{:copy}
 name: Upload Python Package
 
 on:
   release:
-    types: [created]
+    types: [released, prereleased]
```

Comment:
```suggestion
    types: [published]
```

If you save a *prerelease* as a draft and then publish it, you won't get the `prereleased` event.  This doesn't happen for non-pre releases; you just get `released` as usual.  If you just do `published`, you'll get everything.

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release